subxt/src/book/usage/extrinsics.rs

```diff
@@ -173,3 +175,52 @@
 //! Take a look at the API docs for [`crate::tx::TxProgress`], [`crate::tx::TxStatus`] and
 //! [`crate::tx::TxInBlock`] for more options.
 //!
+//! ## Decoding Extrinsics
+//!
+//! Subxt uses a block-centric API to allow users to decode extrinsics. This approach requires users to retrieve
+//! a block's body from the chain before being able to decode the extrinsics. By considering that the block's body
+//! consists of the extrinsics and an additional justification, this API aligns well with the operational nature of the chain
```

Comment:
I think the polkadot docs is a slightly better

> The block body is made up of extrinsics representing the generalization of the concept of transactions. Extrinsics can contain any external data the underlying chain wishes to validate and track.

Mentioning justifications here is a bit hard to follow, it's specific finality proof which is not super important for subxt :)